CHANGES/532.feature

```diff
@@ -0,0 +1,2 @@
+Added a limit of 4MB to non-Blob content, through the `OCI_PAYLOAD_MAX_SIZE` setting, to protect
+against OOM DoS attacks.
```

Comment:
```suggestion
Introduced a size limit for non-Blob content (i.e., OCI manifests/signatures) to prevent OOM DoS attacks during syncing and pushing.
The limit is configurable via the `MAX_MANIFEST_BODY_SIZE` setting, with a default of 4MB.
```

Let's try to be more explicit.